src/components/DataTable/DataTable.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	export let tableData = [{}];
+</script>
+
+{#if tableData.length > 0}
+	<table class="w-full text-sm">
+		<thead class="bg-gray-20">
+			<tr class="w-full h-12 border-b-2">
+				{#each Object.keys(tableData[0]) as columnHeading}
+					<th class="text-01 font-semibold px-4 p-4">{columnHeading}</th>
+				{/each}
+			</tr>
+		</thead>
+		<tbody class="w-full bg-gray-10">
+			{#each Object.values(tableData) as row}
+				<tr class="w-full h-12">
+					{#each Object.values(row) as cell}
+						<td class="p-4 align-middle">{cell}</td>
+					{/each}
+				</tr>
+			{/each}
+		</tbody>
+	</table>
+{/if}
+
+<style lang="scss">
```

Comment:
To use `@apply` without warning you need `lang="postcss"` instead

Reply:
@Th1nkK1D When I add lang="postcss", but it has error.

")" is expected

If you expect this syntax to work, here are some suggestions:
If you use less/SCSS with svelte-preprocess, did you add lang="scss"/lang="less" to your style tag? If you use SCSS, it may be necessary to add the path to your NODE runtime to the setting svelte.language-server.runtime, or use sass instead of node-sass.
Did you setup a svelte.config.js?
See https://github.com/sveltejs/language-tools/tree/master/docs#using-with-preprocessors for more info.svelte(css-syntax-error)

Reply:
Ah my bad, I didn't realise that you use CSS nesting which is a `scss` feature. Let's keep it as it is.